### PR TITLE
Create locale.xml for Spanish translations

### PR DESCRIPTION
**Pull request by gonetil**
Just a few messages  for makeSubmission block plugin.

### CODE REVIEW

**Comment by Gulmork** [comment]
Hi, I have downloaded your es_ES XMLand setup folders needed so that this plug in translates properly into spanish, but In can't seem to get it to display as spanish in my site. I only get a the usual error: ##plugins.block.makeSubmission.linkLabel##

How do I get it to work?

**Comment by gonetil** [comment]
Hi @Gulmork ,
all you have to do is copy the file locale.xml into OJS3_DIR/plugins/blocks/makeSubmission/locale/ dir, and make sure your web server can access it (file permissions). I did nothing but that and it worked like a charm.

File: locale/es_ES/locale.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * locale.xml
+  *
+  * Copyright (c) 2014-2016 Simon Fraser University Library
+  * Copyright (c) 2003-2016 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Localization strings for the es_ES (Spain Spanish) locale.
```
Comment (mtub) [inline locale/es_ES/locale.xml]: The locale's name in PKP applications is "Español (España)", could you change this here and in line 14? (If you cannot update your PR, please let me know and I take care of updating this.)

**Review by mtub — COMMENTED** [review]
Sorry, I didn't see your PR before because the plugin was not on my watch list. Please have a look at my comment and let me know how to proceed. Thanks for contributing!

**Review by mtub — APPROVED** [review]
Thanks!

**Comment by gonetil** [comment]
Glad I helped! And sorry I didn't answer before. It was already hard to edit files using my cel phone :-D

**Pull request merged**